lib/macro/model.rb

```diff
@@ -4,7 +4,7 @@ module Macro
   def self.Model(entity:, connections: [], find_by_key: :id, params_key: :id, assign: false)
     task = ->((ctx, flow_options), **) {
       if assign
-        Macro::Assign(to: :model, path: [*entity, *connections], try: true)[:task].call(ctx)
+        Macro::Assign(to: :model, path: [*entity, *connections], try: true)[:task].call(ctx, **{ options: [] })
```

Comment:
Why are you update this args passing? Please describe.

Reply:
(ruby-2.7)[https://www.ruby-lang.org/en/news/2019/12/25/ruby-2-7-0-released/]